docs/api_v2/api_v2_reference.apib

```diff
@@ -1204,6 +1204,9 @@ Only runs that are starting or running are aborted. For runs with status `FINISH
     + actorId: `janedoe~my-actor` (string, required) - Actor ID or a tilde-separated owner's username and actor name.
     + runId: `3KH8gEpp4d8uQSe8T` (string, required) - Run ID.
     + token: `rWLaYmvZeK55uatRrZib4xbZs` (string, required) - API authentication token.
+    + gracefully: true (boolean, optional)    - If true passed, the actor run will abort gracefully.
+                                                It will send aborting and persistState event into run and force kill run after 30 seconds.
+                                                It is helpful in a case where you plan to resurrect run.
 
```

Comment:
Here, could you use backticks on param/property names to make them stand out? e.g. \`true\`, \`aborting\`, \`persistState\`

Also, maybe we can use less violence? 😄  'force kill run' => 'force-stop the run' ?

Just some pedantic note: 'It is helpful in a case where you plan to resurrect run.' => 'It is helpful in cases where you plan to resurrect the run later.'

Thanks!